css/style.css

```diff
@@ -1,5 +1,127 @@
+* {
+    font-family: 'Righteous', sans-serif;
+    color: white;
+    text-shadow: -1px 0 black, 0 2px black, 2px 0 black, 0 -1px black;
+}
+
+header {
+    height: 160px;
+}
+
+nav {
+    background-color: #6BBAA7;
+    margin: 0 auto;
+    width: 1200px;
+}
+
+nav li {
+    display: inline;
+    margin-left: 110px;
+    font-size: 20px;
+    background-color: #ff8c67;
+    border-radius: 35%;
+    padding: 2px;
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 70px;
+    text-align: center;
+    width: 1200px;
+    margin: 0 auto;
+    height: 60px;
+    padding-bottom: 25px;
+}
+
 body {
-    background-color: gray;
+    background-image: url(../img/seattle-skyline.jpg);
+    background-size: 100%;
```

Comment:
FYI: This can leave a blank space at the bottom of your background below the image. Use `background-size: cover;` to ensure that the entire background is filled by the image.